spec/spec_helper.rb

```diff
@@ -22,7 +23,7 @@
   exit!
 end
 
-Bundler.require(:spec)
+# Bundler.require(:spec)
```

Comment:
@wokenqingtian Wondering why you comment out this line?

Reply:
Because I found Bundler can still require all gems in the spec group without this line. So I comment out it. Any opinion on this?

Reply:
@wokenqingtian good point. I could see https://github.com/jeremytregunna/ruby-trello/blob/e3a57425ed94fc1442af7e20423c5edb11c49497/spec/spec_helper.rb#L19. What do you think about modify that setup line to load just the spec group?

Reply:
I did some tests with it and found the tests can still pass even without `Bundler.setup`. It seems because we have manual require those gems need for testing.
Maybe we can just get rid of this:

```ruby
begin
  ENV['BUNDLE_GEMFILE'] = File.expand_path('../Gemfile', File.dirname(__FILE__))
  require 'bundler'
  Bundler.setup
rescue Bundler::GemNotFound => e
  STDERR.puts e.message
  STDERR.puts 'Try running `bundle install`.'
  exit!
end
```
And replace it with one line code:

```ruby
require "bundler/setup"
```

How do you like it?

Reply:
I see. If so, go for it.